web/auth/ldap/REAMDE.md

```diff
@@ -0,0 +1,26 @@
+# Installation
+
+The LDAP authentication module required the python-ldap package to be available to FAME. It can be installed with this command: `utils/run.sh -m pip install python-ldap~=3.2.0`. This install the required package into the pip environment that is used by FAME.
```

Comment:
```suggestion
The LDAP authentication module requires the python-ldap package to be available to FAME. It can be installed with this command: `utils/run.sh -m pip install python-ldap~=3.2.0`. This installs the required package into the virtualenv that is used by FAME.
```